src/connectors/aws_collect.py

```diff
@@ -459,6 +459,17 @@
         ('target_grants', 'VARIANT'),
         ('target_prefix', 'STRING'),
     ],
+    # https://docs.aws.amazon.com/cli/latest/reference/s3api/get-public-access-block.html
+    's3_get_public_access_block': [
+        ('recorded_at', 'TIMESTAMP_LTZ'),
+        ('account_id', 'STRING'),
+        ('error', 'VARIANT'),
+        ('bucket', 'STRING'),
```

Comment:
sorry for inconsistency above, but "bucket" should go before "error", since it's a parameter not a result